src/test/java/CuidTest.java

```diff
@@ -13,13 +13,29 @@ public void createCuid() throws Exception {
         assertEquals(Cuid.createCuid().charAt(0), 'c');
     }
 
+    @Test
+    public void testForCollisions() {
+        assertEquals(hasNoCollisions(), true);
+    }
+
+    @Test
+    public void shouldValidateACuid() throws Exception {
+        final String cuid = Cuid.createCuid();
+        assertTrue(Cuid.validate(cuid));
+    }
+
+    @Test
+    public void shouldNotAcceptNullCuidWhenValidate() throws Exception {
+        assertFalse(Cuid.validate(null));
+    }
+
```

Comment:
can you add another test case that tests a few strings that are not cuids? Such as 

`hi`, `dh72gsb320000udocl363eofy`, `ch72gsb3200*0udocl363eofy`, `ch72gsb3200p0udocl363eofy`, `ch72gsb3200*0udocl363eof`